setup.py

```diff
@@ -9,5 +9,8 @@
       license='MIT',
       packages=['instrumentserver'],
       zip_safe=False,
-      entry_points={"console_scripts": ["instrumentserver = instrumentserver.apps:serverScript"]}
+      entry_points={"console_scripts": ["instrumentserver = instrumentserver.apps:serverScript",
```

Comment:
these script names are very generic and it's not immediately clear that they belong to the instrumentserver package.
I would prefer naming them 'instrumentserver-dashboard', or something like that.